runtime/engine/src/main/java/io/aklivity/zilla/runtime/engine/config/EngineConfigReader.java

```diff
@@ -190,7 +190,7 @@ private boolean validateAnnotatedSchema(
         validate:
         try
         {
-            final JsonObject annotatedSchemaObject = (JsonObject) annotateJsonObject(schemaObject);
+            final JsonObject annotatedSchemaObject = jsonAnnotator.apply(schemaObject);
```

Comment:
```suggestion
final EngineConfigAnnotator annotator = new EngineConfigAnnotator();
final JsonObject annotatedSchemaObject = annotator.annotate(schemaObject);
```

Given the state needed by internals of annotator, suggest we just instantiate this here on stack and call the annotate method directly instead of having a field.

Reply:
That's what I was thinking but then thought that you have resaon